draft-ietf-tls-esni.md

```diff
@@ -420,6 +420,19 @@ retry_configs
 decreasing order of preference, to be used by the client in subsequent
 connection attempts. These are known as the server's "retry configurations".
 
+Finally, when the client offers the "encrypted_client_hello", if the payload is
+the `inner` variant and the server responds with HelloRetryReqwuest, it MUST
```

Comment:
I'm slightly worried that the "MUST" here violates extensions guidelines.

Reply:
How is this different from, say, requiring the server to send a key_share extension in SH?

Reply:
Not any different, I don't think.